test/main_test.go

```diff
@@ -136,23 +136,29 @@ func configureTestOutput() {
 	opt.Output = io.MultiWriter(opt.Output, mainLogFile)
 }
 
-func FeatureContext(s *godog.Suite) error{
+func FeatureContext(s *godog.Suite) error {
 	// Create kube client
-	if err := framework.InitKubeClient(); err != nil{
+	if err := framework.InitKubeClient(); err != nil {
 		return err
 	}
 
+	// Verify Setup
+	if err := framework.CheckSetup(); err != nil {
+		return err
+	}
```

Comment:
I've added this check setup stage in order to early fail the test.
This stage must be done right after the InitKubeClient is invoked.